dvc/repo/experiments/executor/local.py

```diff
@@ -56,11 +55,4 @@ def _config(self, cache_dir):
     def cleanup(self):
         super().cleanup()
         logger.debug("Removing tmpdir '%s'", self._tmp_dir)
-        try:
-            self._tmp_dir.cleanup()
-        except PermissionError:
-            if os.name == "nt" and sys.version_info < (3, 8):
-                # see https://bugs.python.org/issue26660
-                remove(self._tmp_dir.name)
-                return
-            raise
+        remove(self._tmp_dir.name)
```

Comment:
TemporaryDirectory.cleanup sometimes has issues cleaning up read-only files in git directories on windows. Using our own utils.fs.remove (which handles the permission issues with rmtree(..., onerror=...) fixes the issue